server.php

```diff
@@ -65,9 +65,11 @@ function valet_support_xip_io($domain)
 $valetSitePath = null;
 
 foreach ($valetConfig['paths'] as $path) {
-    if (is_dir($path.'/'.$siteName)) {
-        $valetSitePath = $path.'/'.$siteName;
-
+    $domain = ($pos = strrpos($siteName, '.')) !== false
+                ? substr($siteName, $pos+1)
+                : null;
+    if (is_dir($path.'/'.$siteName) || is_dir($path.'/'.$domain)) {
```

Comment:
I'm using park with non ssl sites, as well as another site served with link. (eg ~/Sites parked, and ~/SomeApp linked).